### PR TITLE
Added Column Sort package.

### PR DESCRIPTION
**Pull request by MatiMax**
Added the Column Sort package for sorting tab-separated text by column.

### CODE REVIEW

File: repository/c.json [hunk 1]
```diff
@@ -1780,6 +1780,16 @@
...
+			"name": "Column Sort",
+			"details": "https://github.com/MatiMax/ColumnSort",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master",
+					"tags": true
+				}
+			]
+		}		{
```
Comment (FichteFoll) [inline repository/c.json]: You forgot a comma and a newline here.

File: repository/c.json [hunk 2]
```diff
@@ -1780,6 +1780,16 @@
 			]
 		},
 		{
+			"name": "Column Sort",
+			"details": "https://github.com/MatiMax/ColumnSort",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master",
```
Comment (FichteFoll) [inline repository/c.json]: You have to remove this line.

**Comment by MatiMax** [comment]
The Column Sort package should now be good to go.

@FichteFoll: Thanks for your support.

**Comment by MatiMax** [comment]
Reopened for merging to head.

**Pull request merged**